pkg/caret/tests/testthat/test_models_lfda.R

```diff
@@ -21,4 +23,14 @@ test_that('test lfda model training and prediction', {
   expect_that(dim(tr_dat)[1], equals(dim(transformed.train)[1]))
 })
 
+test_that('test lfda model visualization function', {
+  skip_on_cran()
+  set.seed(1)
+  tr_dat <- twoClassSim(200)
+
+  lfda.model <- lfda(x=tr_dat[,-16],y=tr_dat[,16],r=3)
```

Comment:
@terrytangyuan Also, I deleted my previous comment, which in retrospect I shouldn't have done.  It read "You need to load the lfda package for this test".